consul/util.go

```diff
@@ -125,6 +130,11 @@ func isConsulServer(m serf.Member) (bool, *serverParts) {
 
 	datacenter := m.Tags["dc"]
 	_, bootstrap := m.Tags["bootstrap"]
+	var disabled uint64 = 0
+	_, disabledStr := m.Tags["disabled"]
```

Comment:
Note to self - where does this get set ever? We probably won't actually propagate this via tags.